src/styles/_sizes.media.scss

```diff
@@ -5,6 +5,42 @@ $medium-width: 1220px;
 $preview-small-width: 849px;
 $preview-medium-width: 1000px;
 
+$size-small-max: 374px;
+$size-medium-min: 375px;
```

Comment:
Minor, but we could use SCSS math operations to halve our variable count.

Reply:
This is no longer needed due to changing the media queries from explicit to overlapping. The number of variables has been consolidated to `$size-small-max`, `$size-medium-max`, and `$size-large-max`.